docs/howto/blueprints.rst

```diff
@@ -0,0 +1,24 @@
+Use Blueprints
```

Comment:
Would we use a title that would be more descriptive of the problem we're typing to solve ?
Something like `Define re-usable tasks in a third party application` ?